deploy/webhook/appliedmanifestwork_webhook.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: appliedmanifestworkvalidators.admission.work.open-cluster-management.io
+webhooks:
+- name: appliedmanifestworkvalidators.admission.work.open-cluster-management.io
+  failurePolicy: Fail
+  clientConfig:
+    service:
+      # reach the webhook via the registered aggregated API
+      namespace: default
+      name: kubernetes
+      path: /apis/admission.work.open-cluster-management.io/v1/appliedmanifestworkvalidators
+  rules:
+  - operations:
+    - CREATE
+    - UPDATE
+    apiGroups:
+    - work.open-cluster-management.io
+    apiVersions:
+    - "*"
+    resources:
+    - appliedmanifestworks
+  admissionReviewVersions: ["v1beta1"]
```

Comment:
Why not use `v1` for `admissionReviewVersions`?

Reply:
yes, i think we could use v1 :) will update that